Dockerfile

```
@@ -17,7 +17,7 @@ RUN wget -qO /opt/tini "https://github.com/krallin/tini/releases/download/v0.18.
     && echo "12d20136605531b09a2c2dac02ccee85e1b874eb322ef6baf7561cd93f93c855 /opt/tini" | sha256sum -c - \
     && chmod +x /opt/tini
 
-ARG BITCOIN_VERSION=0.18.1
+ARG BITCOIN_VERSION=0.21.1
```

```suggestion
ARG BITCOIN_VERSION=0.22
```

Why not upgrade to the last version at this point?

---

Thx for your response.
I have not upgraded to 22.0 because the signature verification process is pretty different.
I can propose a new PR with the last version

---

I don't see any reason to make the dev life more complex, I'm happy with v0.21.1 :) but also I'm happy to review your PR for 22.0 support :)